core/include/cubos/core/geom/box.hpp

```diff
@@ -52,6 +52,16 @@ namespace cubos::core::geom
             corners[6] = {-halfSize.x, halfSize.y, halfSize.z};
             corners[7] = {halfSize.x, -halfSize.y, -halfSize.z};
         }
+
+        void normals(glm::vec3 normals[6]) const
```

Comment:
```suggestion
        static void normals(glm::vec3 normals[6]) const
```

Since this doesn't use the object's data, I think you can make it static.